src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/LegacyCaseRepository.java

```diff
@@ -12,7 +13,7 @@
 
 @RegisterMapper(ClaimMapper.class)
 @SuppressWarnings("squid:S1214") // Pointless to create class for string statement
-public interface ClaimSearchRepository {
+public interface LegacyCaseRepository {
```

Comment:
Why do we need that interface which is a copy of `uk.gov.hmcts.cmc.claimstore.repositories.ClaimRepository`? We should not need to change `uk.gov.hmcts.cmc.claimstore.repositories.ClaimRepository` when moving to CCD beside small changes triggered by API change.

Moving methods from one place to another makes no much sense.

I would scrap `LegacyCaseRepository ` and keep using old DBI repository.

Reply:
not sure other than it makes it clearer whats been moved to CCD, 
I tried to move more but I got a cyclic dependency.

Moving the methods allowed me to see a couple of places where we still weren't using the auth.

I don't mind either way

Reply:
My reasoning is that the code should not be used to track what is left, we have stories for that.

My approach would be extract interface from old repository and start implementing the new one, non implemented methods would throw `NotImplementedException` exception.

Anyway nothing to do with this PR, however I would stop moving things between two repos that play the same role.